src/types/debt_offer_data.ts

```diff
@@ -0,0 +1,22 @@
+import { BigNumber } from "../../utils/bignumber";
+
+import { ECDSASignature } from "./ecdsa_signature";
+
+/**
+ * Contains the terms, using types compatible for sending to a blockchain contract, for which a
+ * creditor would be willing to enter into a debt agreement.
+ */
+export interface DebtOfferData {
```

Comment:
I think we also need to include other things, like repayment router, creditor fee, underwriter, underwriter risk rating, terms contract, terms contract parameters, salt